examples/rest-api-client-demo/src/space.ts

```diff
@@ -83,6 +83,15 @@ export class Space {
     }
   }
 
+  public async addThread() {
+    const name = "Added Thread Name";
```

Comment:
This would be easier to understand when checking on the kintone screen if it were changed as follows!

```suggestion
    const name = "The thread added via rest-api-client";
```

Reply:
addressed it in f0dceaac121df00fce20e8bbf53af50393ff9d9b